File: ms/blueprintsprocessor/modules/inbounds/health-api-common/src/test/kotlin/org/onap/ccsdk/cds/blueprintsprocessor/healthapi/SecurityConfigurationTest.java
package org.onap.ccsdk.cds.blueprintsprocessor.healthapi;

import org.junit.Assert;
import org.junit.Test;
import org.onap.ccsdk.cds.blueprintsprocessor.healthapi.configuration.SecurityEncryptionConfiguration;
import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;

public class SecurityConfigurationTest {

  @Test public void testEncryption() {
    String passWord = "ccsdkapps";
    SecurityEncryptionConfiguration securityCOnfiguration = new SecurityEncryptionConfiguration();
    String encryptedValue = securityCOnfiguration.encrypt(passWord);
    System.out.println(encryptedValue);
    Assert.assertEquals(passWord, securityCOnfiguration.decrypt(encryptedValue));
  }



}

File: ms/blueprintsprocessor/modules/inbounds/health-api-common/src/test/kotlin/org/onap/ccsdk/cds/blueprintsprocessor/healthapi/SecurityConfigurationTest.kt
package org.onap.ccsdk.cds.blueprintsprocessor.healthapi

import org.junit.Assert
import org.junit.Test
import org.onap.ccsdk.cds.blueprintsprocessor.healthapi.configuration.SecurityEncryptionConfiguration


class SecurityConfigurationTest {

    @Test
    fun testEncryption() {
        val passWord = "ccsdkapps"
        val securityConfiguration = SecurityEncryptionConfiguration()
        val encryptedValue = securityConfiguration.encrypt(passWord)
        println(encryptedValue)
        Assert.assertEquals(passWord, securityConfiguration.decrypt(encryptedValue!!))
    }
}
